contracts/WstETHPriceFeed.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "./vendor/@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+interface IWstETH {
+    function decimals() external view returns (uint8);
+    function tokensPerStEth() external view returns (uint256);
+}
+
+contract WstETHPriceFeed is AggregatorV3Interface {
+    /** Custom errors **/
+    error InvalidInt256();
+    error NotImplemented();
+
+    string public constant override description = "Custom price feed for wstETH / USD";
+
+    uint public constant override version = 1;
+
+    /// @notice Scale for returned prices
+    uint8 public override decimals = 8;
+
+    /// @notice Chainlink stETH / USD price feed
+    address public immutable stETHtoUSDPriceFeed;
+
+    /// @notice WstETH contract address
+    address public immutable wstETH;
+
+    /// @notice scale for WstETH contract
+    uint public immutable wstETHScale;
+
+    constructor(address stETHtoUSDPriceFeed_, address wstETH_) {
```

Comment:
So basically we don't need the wstETH/stETH price feed? Interesting that it exists

Reply:
Is there one for mainnet?

I only see `wstETH / stETH` for [Optimism](https://data.chain.link/optimism/mainnet/crypto-eth/wsteth-steth%20exchangerate) and [Arbitrum](https://data.chain.link/arbitrum/mainnet/crypto-eth/wsteth-steth%20exchangerate).

Looks like they're returning the same value as [wstETH.stEthPerToken](https://etherscan.io/token/0x7f39c581f595b53c5cb19bd0b3f8da6c935e2ca0#readContract)

Reply:
You're right I don't see Ethereum mainnet